Assets/MixedRealityToolkit.ThirdParty/MRTK-Quest/Scripts/Config/MRTKOculusConfig.cs

```diff
@@ -106,10 +108,25 @@ public static MRTKOculusConfig Instance
         [Tooltip("Custom hand material to use for hand tracking hand mesh.")]
         private Material customHandMaterial = null;
 
+        public System.Action onCustomHandMaterialUpdate;
         /// <summary>
         /// Custom hand material to use for hand tracking hand mesh.
         /// </summary>
-        public Material CustomHandMaterial => customHandMaterial;
+        public Material CustomHandMaterial
+        {
+            get
+            {
+                return customHandMaterial;
+            }
+
+            set
+            {
+                customHandMaterial = value;
+                if (onCustomHandMaterialUpdate != null)
```

Comment:
you can do a ternary null check for events like this.
onCustomHandMaterialUpdate?.Invoke() I believe

Reply:
Thank you for the suggestion. 
Updated the requested change